index.js

```diff
@@ -4,11 +4,16 @@
 const { extname } = require('path');
 
 const config = hexo.config.sitemap = Object.assign({
-  path: 'sitemap.xml'
+  path: 'sitemap.xml',
+  rel: true
```

Comment:
IMHO, default config should be 'false', since sitemap can be also declared in robots.txt which makes insert tag in HTML unnecessary.

Reply:
well yeah, declaring it in robots.txt is one of current official ways, whereas rel-sitemap is still in proposal (MDN currently [doesn't list](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types) it). I figure no harm adding it by default. Anyhow, I'm fine with disabled by default as well.